### PR TITLE
Fix HGVS refseq transcripts

### PR DESCRIPTION
**Pull request by dglemos**
This is a fix for e!96 
#385 is for e!97

### CODE REVIEW

**Comment by sarahhunt** [comment]
Thanks @dglemos - can you add a test?

**Comment by ens-lgil** [comment]
@sarahhunt, is it ok to merge it ?

File: modules/Bio/EnsEMBL/Variation/TranscriptVariationAllele.pm [hunk 1]
```diff
@@ -1023,7 +1023,10 @@ sub hgvs_protein {
   ## checks complete - start building term
 
   ### get reference sequence and add seq version unless LRG
-  $hgvs_notation->{ref_name} = $tr->translation->display_id();
+  my $tr_id = $tr->translation->display_id();
+  $tr_id =~ s/.*NP_/NP_/;
```
Comment (sarahhunt) [inline modules/Bio/EnsEMBL/Variation/TranscriptVariationAllele.pm]: This won't fix all records, see cds-NP_001185471.1-2.1:p.Asn46Ser here :

http://rest.ensembl.org/variant_recoder/human/rs764041571?content-type=application/json

Does something like this work:

https://github.com/Ensembl/ensembl-vep/blob/release/96/modules/Bio/EnsEMBL/VEP/AnnotationSource/Database/Transcript.pm#L668

Comment (dglemos) [reply modules/Bio/EnsEMBL/Variation/TranscriptVariationAllele.pm]: I thought the lazy fix would work. It's fixed now.

**Pull request merged**